### PR TITLE
Support for next lap timer

### PR DESCRIPTION
**Pull request by error414**
This is configuration for cleanflight/cleanflight#2580

### CODE REVIEW

**Comment by hydra** [comment]
this needs updating now that we have changed MSP_TRANSPONDER_CONFIG.

**Comment by hydra** [comment]
Basic support for ILAP via API 1.33.0 added in https://github.com/cleanflight/cleanflight-configurator/pull/450

File: js/msp/MSPCodes.js [hunk 1]
```diff
@@ -71,8 +71,8 @@ var MSPCodes = {
     MSP_SET_PID_ADVANCED:           95,
     MSP_SENSOR_CONFIG:              96,
     MSP_SET_SENSOR_CONFIG:          97,
-    // MSP_SPECIAL_PARAMETERS:         98, removed
-    //MSP_SET_SPECIAL_PARAMETERS:     99, removed
+	MSP_TRANSPONDER_TYPE:           98,
+	MSP_SET_TRANSPONDER_TYPE:       99,
```
Comment (hydra) [inline js/msp/MSPCodes.js]: need to remove this

File: js/msp/MSPHelper.js [hunk 2]
```diff
@@ -875,7 +875,7 @@ MspHelper.prototype.process_data = function(dataHandler) {
                 console.log("Blackbox config saved");
                 break;
             case MSPCodes.MSP_TRANSPONDER_CONFIG:
```
Comment (hydra) [inline js/msp/MSPHelper.js]: MSP commands already updated in the other PR.

File: js/msp/MSPHelper.js [hunk 3]
```diff
@@ -885,6 +885,9 @@ MspHelper.prototype.process_data = function(dataHandler) {
             case MSPCodes.MSP_SET_TRANSPONDER_CONFIG:
                 console.log("Transponder config saved");
                 break;
+			case MSPCodes.MSP_SET_TRANSPONDER_TYPE:
```
Comment (hydra) [inline js/msp/MSPHelper.js]: remove this

File: js/msp/MSPHelper.js [hunk 4]
```diff
@@ -1121,6 +1124,10 @@ MspHelper.prototype.crunch = function(code) {
             }
             break;
 
+		case MSPCodes.MSP_SET_TRANSPONDER_TYPE:
```
Comment (hydra) [inline js/msp/MSPHelper.js]: remove this

File: tabs/transponder.js [hunk 5]
```diff
@@ -1,104 +1,211 @@
 'use strict';
 
-TABS.transponder = {
-    available: false
+var transponderType = {
```
Comment (hydra) [inline tabs/transponder.js]: remove this, see TRANSPONDER.providers = you can find an i18n message for each provider using the id in the i18n message key.  e.g. 'transponderProvider0', 'transponderProvider1', etc.  There should be NO ilap or arcitimer specific code required when the new MSP_TRANSPONDER_CONFIG is used.

**Comment by hydra** [comment]
the UI side of things needs updating, rebase this PR on #450

**Comment by error414** [comment]
https://github.com/cleanflight/cleanflight-configurator/pull/452

**Pull request closed without merging**